config/vufind/config.ini

```diff
@@ -2292,6 +2292,24 @@ legacyPbkdf2 = true
 ; Default file_permission seems to be 0600.
 ;file_permission = 0600
 
+; This section controls the settings for caching of configuration.
+; Please note that all caching settings will not be affected by the configuration caching it self
+; and will always be loaded without caching.
+[ConfigCache]
+cacheName = "config"
+reloadOnFileChange = true
+disabled = false
+
+; It is possible to override the config cache settings for specific configurations handlers.
+; By default the INI configuration caching is disabled.
+[CacheConfigHandler_ini]
+disabled = true
+
+; It is possible to override the config cache settings for specific config names.
+; By default the searchspecs configuration caching is disabled.
```

Comment:
Maybe "if unset below" would be more clear than "by default" in this sentence since the default setting in this file contradicts the default behavior if there is no setting.